src/client/apps/assistant/components/messages.tsx

```diff
@@ -29,6 +30,9 @@ export function Messages({
 	message: string;
 	color: ColorPaletteProp;
 }) {
+	useEffect(() => {
+		console.log(message, messages);
```

Comment:
Do we really need a useless effect here? Totally fine if you want to keep this for debugging 🙏